README.md

```diff
@@ -190,9 +191,9 @@ version: '3.8'
 services:
   mailserver:
     image: docker.io/mailserver/docker-mailserver:latest
-    hostname: mail          # ${HOSTNAME}
-    domainname: domain.com  # ${DOMAINNAME}
-    container_name: mail    # ${CONTAINER_NAME}
+    hostname: mail
+    domainname: domain.com
```

Comment:
`domain.com` is a valid registered domain. We shouldn't use that as an example.

Instead, `example.com` should be used, which is defined exactly for that [use case](https://www.iana.org/domains/reserved)

Reply:
Agree.